index.html

```diff
@@ -289,6 +289,80 @@ <h2>About me</h2>
           </ul>
         </div>
       </div>
+
+      <form class="contact-form">
+        <h2>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
+
+        <input type="text" placeholder="Full name" />
+        <input type="text" placeholder="Email address"/>
```

Comment:
Kindly consider adding the correct type attribute to the EMAIL FIELD so users can not violate the process and you only receive the authenticated data. Moreover, there needs a restriction so a user can't leave the EMAIL Input empty.  To achieve all the above you can use the following piece of code as an example.

`<input type="email" placeholder="Email address" required/>`